index.html

```diff
@@ -8,4 +8,6 @@
     <body>
         <h1>Section 2: JavaScript Language Basics</h1>
     </body>
-</html>
\ No newline at end of file
+
+    <script src='script.js'></script>
```

Comment:
script tags are included inside the `body` tag.

Reply:
Fixed.